core/storage/rocksdb/rocksdb.cpp

```diff
@@ -118,28 +103,28 @@ namespace kagome::storage {
   void RocksDb::dropColumn(kagome::storage::Space space) {
     auto space_name = spaceName(space);
     auto column_it =
-        std::find_if(cf_handles_.begin(),
-                     cf_handles_.end(),
-                     [&space_name](const ColumnFamilyHandle &handle) {
-                       return handle.name == space_name;
+        std::find_if(column_family_handles_.begin(),
+                     column_family_handles_.end(),
+                     [&space_name](const ColumnFamilyHandlePtr &handle) {
+                       return handle->GetName() == space_name;
                      });
-    if (cf_handles_.end() == column_it) {
+    if (column_family_handles_.end() == column_it) {
       throw DatabaseError::INVALID_ARGUMENT;
     }
-    auto &column = *column_it;
+    auto &handle = *column_it;
     auto e = [this](rocksdb::Status status) {
       if (!status.ok()) {
         logger_->error("DB operation failed: {}", status.ToString());
         throw status_as_error(status);
       }
     };
-    e(db_->DropColumnFamily(column.handle));
-    e(db_->DestroyColumnFamilyHandle(column.handle));
-    e(db_->CreateColumnFamily({}, column.name, &column.handle));
+    e(db_->DropColumnFamily(handle));
+    e(db_->DestroyColumnFamilyHandle(handle));
+    e(db_->CreateColumnFamily({}, handle->GetName(), &handle));
```

Comment:
```suggestion
    auto column_name = handle->GetName();
    e(db_->DropColumnFamily(handle));
    e(db_->DestroyColumnFamilyHandle(handle));
    e(db_->CreateColumnFamily({}, column_name, &handle));
```

Reply:
Why? After calling CreateColumnFamily argument *name* will be already instantiated and will not be depended on source (*handle*).